src/features/isanteplus/patientCoversheet.feature

```diff
@@ -5,8 +5,5 @@ Feature: Patient Cover sheet
 
   @patientCoverSheet
   Scenario: Patient Cover sheet
-  When   Search for and select Patient "<patientName>"
+  When   Search for and select Patient
   Then   Selected patient’s ‘Cover Sheet’ will be displayed with all the right details
-   Examples:
```

Comment:
sorry i think i missed this , 
we are meant to pss values via this data table. can you fix this anyhere you removed the data table ??